packages/contentstack-bootstrap/package.json

```diff
@@ -17,16 +17,16 @@
     "test:report": "nyc --reporter=lcov mocha \"test/**/*.test.js\""
   },
   "dependencies": {
-    "@contentstack/cli-cm-seed": "^1.2.0",
-    "@contentstack/cli-command": "^1.1.0",
-    "@contentstack/cli-utilities": "^1.1.0",
+    "@contentstack/cli-cm-seed": "^1.3.0",
+    "@contentstack/cli-command": "^1.2.0",
+    "@contentstack/cli-utilities": "^1.2.0",
     "@contentstack/management": "^1.6.1",
+    "@oclif/core": "^2.3.0",
```

Comment:
@aman19K If we use @oclif/core from the utility package we can remove the dependency from package.json right.?

Reply:
@antonyagustine removed @oclif/core dependency from package.json